update cli docs (#4020)
langchain-ai/docs: src/langsmith/studio.mdx

Chat mode is a simpler UI for iterating on and testing chat-specific agents. It is useful for business users and those who want to test overall agent behavior. Chat mode is only supported for graph's whose state includes or extends [`MessagesState`](/oss/langgraph/use-graph-api#messagesstate).

## Deploy from Studio

> Go from [testing graphs locally](/langsmith/local-dev-testing) in Studio to deployed on Langsmith Cloud in 1 Click, directly from Studio. This can be used to create a brand new deployment for quick prototyping or to redeploy an existing deployment.

Reviewer: Avoid passive voice to make clear who is acting / speak to the user.
```suggestion
Go from [testing graphs locally](/langsmith/local-dev-testing) in Studio to deploying them on Langsmith Cloud in 1 Click, directly from Studio. You can use this to create a brand new deployment for quick prototyping or to redeploy an existing deployment.
```
